cypress/features/regression/showEditPodClassic.feature

```diff
@@ -1,155 +1,17 @@
-Feature: Show Edit Pod component
-  I want to test Show Edit Pod component
+Feature: Show Edit Pod classic component
+  I want to test Show Edit Pod classic component
 
```

Comment:
Could we also add test to check color for edit icon?

Reply:
fixed